src/NServiceBus.AcceptanceTests/Pipeline/When_message_processing_fails.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace NServiceBus.AcceptanceTests.Pipeline
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_message_processing_fails : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_provide_additional_exception_information()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<EndpointWithFailingHandler>(e => e
+                    .When(s => s.SendLocal(new FailingMessage()))
+                    .DoNotFailOnErrorMessages())
+                .Done(c => c.MessageFailed)
+                .Run(TimeSpan.FromSeconds(10));
```

Comment:
Is TimeSpan necessary given that the exception will be thrown? And if not, the test will time out and fail.

Reply:
no, not necessary, forgot to remove that after writing the test